Use cmake's idea of the "make" program, so that it can be tuned by robotpkg
https://github.com/jrl-umi3218/jrl-cmakemodules/issues/34

--- cmake/man.cmake~	2013-09-25 09:17:12.000000000 +0200
+++ cmake/man.cmake	2013-12-14 19:24:24.000000000 +0100
@@ -43,7 +43,7 @@
     DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${NAME}.1)
 
   # Trigger man page generation at install.
-  INSTALL(CODE "EXECUTE_PROCESS(COMMAND make man)")
+  INSTALL(CODE "EXECUTE_PROCESS(COMMAND ${CMAKE_MAKE_PROGRAM} man)")
 
   # Detects if PKGMAN has been specified
   SET(DESTINATION_MAN_PAGE share/man/man1)
